wp-hooks-docs/src/theme/Footer/index.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+
+import clsx from 'clsx';
+import { ThemeClassNames } from '@docusaurus/theme-common';
+import FooterCopyright from '@theme/Footer/Copyright';
+import styles from './footer.module.css';
+import Link from '@docusaurus/Link';
+
+/**
+ * Get started section component
+ *
+ * @return {JSX.Element} The get started section component
+ */
+function GetStartedSection() {
+	return (
+		<section
+			className={ clsx( styles.getStartedSection, styles.greySection ) }
+		>
+			<div className="container">
+				<h2 className={ styles.sectionTitle }>Ready to Get Started?</h2>
+				<p className={ styles.sectionDescription }>
+					Distributor is a WordPress plugin that makes it easy to
+					syndicate and reuse content across your websites — whether
+					in a single multisite or across the web.
+				</p>
+				<div className={ styles.getStartedActions }>
+					<Link
+						className="button button--primary button--lg"
+						to="https://distributorplugin.com/#cta"
+					>
+						Download Distributor Plugin
+					</Link>
+				</div>
+			</div>
+		</section>
+	);
+}
+
+function Footer() {
+	const currentYear = new Date().getFullYear().toString();
+
+	return (
+		<>
+			<GetStartedSection />
+			<footer
+				className={ clsx(
+					ThemeClassNames.layout.footer.container,
+					styles.mainFooter,
+					'footer'
+				) }
+			>
+				<span
+					aria-hidden="true"
+					className={ styles.footerBackground }
+				></span>
+				<div className="container">
+					<div className={ styles.footerContainer }>
+						<div className={ styles.footerCopyright }>
+							<div className={ styles.footerCopyrightLinks }>
+								<FooterCopyright
+									copyright={ `Finely crafted by Fueled, ©${ currentYear }` }
+								/>
+								<a href="https://github.com/10up/distributor/issues/">
+									Support
+								</a>
```

Comment:
This kind of reads as a donation request as is.

```suggestion
								<a href="https://github.com/10up/distributor/issues/">
									Issues
								</a>
```